components/NavBar/NavBar.tsx

```diff
@@ -61,22 +39,36 @@ const HeaderContentContainer = styled(Container)`
     margin-right: 5rem;
   }
 `
+const TopBarContainer = styled(TopBar)`
+  width: 100%;
+  padding-bottom: 4rem;  
 
+`
 const LogoContainer = styled.div`
   width: fit-content;
   white-space: nowrap;
 `
 
 const ActionsContainer = styled(TopBar.Actions)`
+  width: 100%;
+  background-color: ${tokens.colors.ui.background__default.hex};
   display: flex;
+  justify-content: flex-end;
+
+`
+
+const SearchContainer = styled(TopBar.CustomContent)`
+  width: 80%;
+  background-color: ${tokens.colors.ui.background__default.hex};
```

Comment:
shouldn't need a `background-color` here either

Reply:
i have removed all styling and it works fine . Just looking for sticky thing as that one having some issue

Reply:
sticky is built-in to `TopBar`, so it should work if `TopBar` is the top most parent